### DISCUSSION TITLE
Incorrect Odometry position with Realsense D435 camera

### DISCUSSION THREAD

**Discussion by sandeepdutta** [stella-cv/stella_vslam#593] [Q&A] [answered]
Helllo All,

I am using an Intel Relasense D435 in RGBD mode with stella_vslam_ros. The Odometry position reported seems to 
less by a factor of almost 10 i.e. when it moves 0.1 Meter (wheel odometry), the visual odometry reports a movement of 0.01 Meter. Any suggestions would be greatly appreciated.
Here is my Camera setup.
Camera:
  name: "Realsens D435 in RGBD mode"
  setup: "RGBD"
  model: "perspective"

  fx: 610.543579101562
  fy: 610.674682617188
  cx: 419.73681640625
  cy: 242.103820800781

  k1: 0.0
  k2: 0.0
  p1: 0.0
  p2: 0.0
  k3: 0.0

  fps: 60.0
  cols: 848
  rows: 480
  color_order: "RGB"

Thank you
Sandeep

**Comment by ymd-stella** [comment, marked as answer]
Set Processing::depthmap_factor to 1000.0

https://stella-cv.readthedocs.io/en/latest/parameters.html#preprocessing

example: https://github.com/stella-cv/stella_vslam/blob/0a1f13e2b2ee3f63a8dc40601e6e5d59a4663111/example/tum_rgbd/TUM_RGBD_rgbd_3.yaml#L38-L40

**Reply by sandeepdutta** [reply]
Thank you that worked 
Sandeep
